baseline.gradle

```diff
@@ -52,6 +52,7 @@ subprojects {
   apply plugin: 'com.palantir.baseline-exact-dependencies'
   apply plugin: 'com.diffplug.spotless'
 
+  String scalaVersion = System.getProperty("scalaVersion") != null ? System.getProperty("scalaVersion") : System.getProperty("defaultScalaVersion")
```

Comment:
nit: We could call `getProperty(String key, String def)` instead `System.getProperty("scalaVersion", System.getProperty("defaultScalaVersion"))`. Feel free to ignore this comment as pre-existing.